lib/commons/dom/is-visible.js

```diff
@@ -56,8 +56,7 @@ dom.isVisible = function(el, screenReader, recursed) {
 
 	if (
 		style.getPropertyValue('display') === 'none' ||
-		nodeName.toUpperCase() === 'STYLE' ||
-		nodeName.toUpperCase() === 'SCRIPT' ||
+		['STYLE', 'SCRIPT', 'NOSCRIPT', 'TEMPLATE'].includes(nodeName) ||
```

Comment:
Is `nodeName` always uppercase?

Reply:
@marcysutton - the line above does the uppercasing already - https://github.com/dequelabs/axe-core/pull/1257/files#diff-a871adbd9186fdb557c3a1171796b6d8R55